CardScan/Classes/CreditCardOcr/OcrMainLoop.swift

```diff
@@ -138,16 +129,7 @@ open class OcrMainLoop : MachineLearningLoop {
     }
     
     static func warmUp() {
-        let mainLoop = OcrMainLoop()
-        let image = UIImage.grayImage(size: CGSize(width: 600, height: 600))
-        let roiRectangle = CGRect(x: 0, y: 0, width: 600, height: 600)
-        guard let cgImage = image?.cgImage else { return }
-        for ocr in mainLoop.analyzerQueue {
-            ocr.dispatchQueue.async {
-                let _ = mainLoop
-                let _ = ocr.recognizeCard(in: cgImage, roiRectangle: roiRectangle)
-            }
-        }
+        // TODO(stk): Implement this later
```

Comment:
Sanity check: Should this be resolved before meeting this PR?

Reply:
No, this is all going to change a lot after Jaime finishes up dynamic model downloading, so we'll defer until then.